lib/main.coffee

```diff
@@ -93,35 +90,15 @@ module.exports =
 
   # Public: Creates AutocompleteManager instances for all active and future editors (soon, just a single AutocompleteManager)
   activate: ->
-    @getAutocompleteManager()
-    # @activateTimeout = setTimeout(@getAutocompleteManager, 0)
+    run = =>
+      @getAutocompleteManager()
+    setTimeout(run.bind(this), 0)
```

Comment:
Not really, simply requiring AutocompleteManager / ProviderManager causes the slowdowns - also, FuzzyProvider builds its word list on activation.

The unspying / respying of setTimeout occurs because atom spys and replaces the implementation of setTimeout, which prevents it from functioning correctly.